tests/timing_util_test.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from perfkitbenchmarker import flags_validators
+from perfkitbenchmarker import timing_util
+
+
+class ValidateMeasurementsFlagTestCase(unittest.TestCase):
+  """Tests exercising ValidateMeasurementsFlag."""
+
+  def testEmptyList(self):
+    """Passing an empty list is not allowed."""
+    e = None
+    try:
+      timing_util.ValidateMeasurementsFlag([])
+    except flags_validators.Error as e:
+      pass
+    self.assertIsNotNone(e)
+    self.assertEqual(str(e), 'option --timing_measurements requires argument')
+
+  def testInvalidValue(self):
+    """Passing an unrecognized value is not allowed."""
+    e = None
+    try:
+      timing_util.ValidateMeasurementsFlag(['TEST'])
+    except flags_validators.Error as e:
+      pass
+    self.assertIsNotNone(e)
+    self.assertEqual(str(e), 'TEST: Invalid value for --timing_measurements')
+
+  def testNoneWithAnother(self):
+    """Passing NONE with another value is not allowed."""
+    e = None
+    try:
+      timing_util.ValidateMeasurementsFlag(['NONE', 'RUNTIMES'])
+    except flags_validators.Error as e:
+      pass
+    self.assertIsNotNone(e)
+    self.assertEqual(
+        str(e), 'NONE: Cannot combine with other --timing_measurements options')
+
+  def testValid(self):
+    """Test various valid combinations."""
+    validate = timing_util.ValidateMeasurementsFlag
+    e = None
+    try:
+      self.assertEqual(validate(['NONE']), True)
```

Comment:
I think assertIs(value, True) would be most appropriate here. assertTrue and assertEqual both have behaviors where non-True values pass (ex: ["non-empty list"] for assertTrue, or 1 for assertEqual).